{{ cookiecutter.package_name }}/.travis.yml

```
@@ -127,6 +127,17 @@ matrix:
           stage: Initial tests
           env: TOXENV=codestyle
 
+        # Run documentation link check in a cron job.
+        - language: python
+          python: 3.7
```

Given Python 3.9 is out now, probably okay to bump this.
```suggestion
          python: 3.8
```